hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp.java

```diff
@@ -6,29 +6,42 @@
 import io.kubernetes.client.extended.controller.Controller;
 import io.kubernetes.client.extended.controller.ControllerManager;
 
-import com.linkedin.hoptimator.models.V1alpha1KafkaTopic;
-import com.linkedin.hoptimator.models.V1alpha1KafkaTopicList;
 import com.linkedin.hoptimator.models.V1alpha1Subscription;
 import com.linkedin.hoptimator.models.V1alpha1SubscriptionList;
 import com.linkedin.hoptimator.models.V1alpha1SqlJob;
 import com.linkedin.hoptimator.models.V1alpha1SqlJobList;
-import com.linkedin.hoptimator.operator.flink.FlinkSqlJobReconciler;
-import com.linkedin.hoptimator.operator.kafka.KafkaTopicReconciler;
 import com.linkedin.hoptimator.operator.subscription.SubscriptionReconciler;
 import com.linkedin.hoptimator.planner.HoptimatorPlanner;
 
 import java.io.IOException;
-import java.util.concurrent.TimeUnit;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Properties;
+import java.util.concurrent.TimeUnit;
 
 public class HoptimatorOperatorApp {
 
+  final String modelPath;
+  final String namespace;
+  final Properties properties;
+
+  /** This constructor is likely to evolve and break. */
+  public HoptimatorOperatorApp(String modelPath, String namespace, Properties properties) {
+    this.modelPath = modelPath;
+    this.namespace = namespace;
+    this.properties = properties;
+  }
+
   public static void main(String[] args) throws Exception {
     if (args.length != 1) {
       throw new IllegalArgumentException("Missing model file argument.");
     }
-    HoptimatorPlanner.Factory plannerFactory = HoptimatorPlanner.Factory.fromModelFile(args[0],
-        new Properties());
+    new HoptimatorOperatorApp(args[0], "default", new Properties()).run();
```

Comment:
Is it useful to allow overrides from the command line? If `new Properties()` is replaced with `System.getProperties()`, then `-Dhoptimator.key=value` settings would be honored with `run.sh`. Execution would be similar to:
```sh
$ JAVA_OPTS=-Dhoptimator.cli=value hoptimator-operator/run.sh <model>
```

This might help address the `namespace` TODO.

Reply:
I'd rather keep a distinction between JAVA_OPTS and app options. I opened issue #17 to add `--properties key=value` or similar at some point.